### PR TITLE
changed is_verified to accept null values

### PR DESCRIPTION
**Pull request by geekNero**

### CODE REVIEW

File: detect_secrets/plugins/base.py [hunk 1]
```diff
@@ -67,9 +67,14 @@ def analyze_line(
...
-                    is_verified = True if verified_result == VerifiedResult.VERIFIED_TRUE else False
+                    if verified_result == VerifiedResult.VERIFIED_TRUE:
+                        is_verified = True
+                    elif verified_result == VerifiedResult.VERIFIED_FALSE:
+                        is_verified = False
+                    else:
+                        is_verified = None
                 except requests.exceptions.RequestException:
-                    is_verified = False
+                    is_verified = None
```
Comment (mpurusottamc) [inline detect_secrets/plugins/base.py:77]: we should not return None. It should be False. If we are not able to verify, then it should be False.

**Review by mpurusottamc — CHANGES_REQUESTED** [review]
let's not return None for is verified.

File: detect_secrets/plugins/base.py (hunk 1 above)
Comment (geekNero) [reply detect_secrets/plugins/base.py:77]: The none represents that the verification failed and we have no information whether the secret is valid or not

**Review by mpurusottamc — APPROVED** [review]
makes sense now.

**Pull request merged**